sdk/core/core-tracing/samples-dev/basicTracing.ts

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/**
+ * @summary Uses `@azure/core-tracing` APIs to instrument method calls for a fake Azure client library.
+ */
+
+import { createTracingClient, OperationTracingOptions, TracingClient } from "@azure/core-tracing";
+
+/**
+ * Represents common operation options
+ */
+interface OperationOptions {
+  tracingOptions?: OperationTracingOptions;
+}
+
+/**
+ * Represents an Azure Client that does work against an Azure service.
+ */
+class BasicClient {
+  private tracingClient: TracingClient;
+
+  constructor() {
+    // Create a tracing client in the constructor, passing to it
+    // the package name, the value for az.namespace attribute, and the
+    // optional package version.
+    this.tracingClient = createTracingClient({
+      packageName: "@azure/core-tracing",
+      namespace: "Microsoft.Core",
```

Comment:
perhaps worth linking this again here: https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/azure-services-resource-providers